internal/core/api.rs

```diff
@@ -1025,3 +1025,47 @@ impl std::error::Error for PlatformError {
 fn error_is_send() {
     let _: Box<dyn std::error::Error + Send + Sync + 'static> = PlatformError::NoPlatform.into();
 }
+
+// Sorry, can't test with rust test harness and multiple threads.
```

Comment:
maybe this should be an "integration" test instead?
I was thinking perhaps there should be one test on each renderer

Reply:
Yeah, that would be better. So maybe a duplicated test in `internal/renderers/*/tests/grab_window.rs` ?